recipes/sfml/all/test_package/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.24)
```

Comment:
Building SFML requires 3.22 (or 3.24 on Windows as mentioned above). Using SFML doesn't require anything particularly new. 3.1 is probably too old but something like 3.10 or 3.16 is probably sufficient. 3.16 or 3.22 are reasonable choices since it's rare you find an OS still in use today that ships with anything older than that.

Reply:
This is just the test package rather than what's shipped. But for the main package, you're right, it should help those using --build=always/missing.

Reply:
Package part fixed in a6b8e39528c02a2aa8f3397f7be3ba7814d9159c